compiler/riscv/codegen/RVInstruction.hpp

```diff
@@ -1298,6 +1298,65 @@ class AdminInstruction : public TR::Instruction
    virtual int32_t estimateBinaryLength(int32_t currentEstimate);
    };
 
+class DataInstruction : public TR::Instruction
+   {
+   uint32_t _sourceImmediate;
+
+public:
+
+   /*
+    * @brief Constructor
+    * @param[in] op : instruction opcode
+    * @param[in] node : node
+    * @param[in] imm : immediate value
+    * @param[in] cg : CodeGenerator
+    */
+   DataInstruction(TR::InstOpCode::Mnemonic op, TR::Node *node, uint32_t imm, TR::CodeGenerator *cg)
+      : TR::Instruction(op, node, cg), _sourceImmediate(imm)
+      {
+      TR_ASSERT(op == TR::InstOpCode::dd, "Opcode of data instruction is not dd");
```

Comment:
We should use `TR_ASSERT_FATAL_WITH_NODE` here. For future we are trying to avoid `TR_ASSERT` moving forward since they are only active in debug builds which we don't currently build or test. Similarly elsewhere.

Reply:
I see. Fixed in 0f81a4f